closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -1108,18 +1108,22 @@ JNIEXPORT jint JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_CBCInit
         return -1;
     }
 
-    switch(key_len) {
-        case 16:
-            evp_cipher1 = (*OSSL_aes_128_cbc)();
-            break;
-        case 24:
-            evp_cipher1 = (*OSSL_aes_192_cbc)();
-            break;
-        case 32:
-            evp_cipher1 = (*OSSL_aes_256_cbc)();
-            break;
-        default:
-            break;
+    if (JNI_TRUE == doReset) {
+        evp_cipher1 = NULL;
+    } else {
+        switch(key_len) {
```

Comment:
`evp_cipher1` is already `NULL` here, booleans should only be compared to false (perhaps implicitly).
Please also add the missing space after `switch`.
```c
    if (!doReset) {
        switch (key_len) {
```

Reply:
Made this update to check for JNI_FALSE and not set evp_cipher1 to NULL twice.